.github/workflows/node.js.yml

```
@@ -38,6 +38,7 @@ jobs:
     - run: npm ci
     - run: npm install cypress-mochawesome-reporter
     - run: npx sequelize-cli db:migrate
+    - run: npx eslint . --fix
```

Let's do this step before running db:migrate - fail fast.